server/sonar-process/src/main/java/org/sonar/process/logging/LogbackHelper.java

```diff
@@ -268,57 +243,13 @@ private static Level getPropertyValueAsLevel(Props props, String propertyKey) {
 
   private static boolean isAllowed(Level level) {
     for (Level allowedRootLogLevel : ALLOWED_ROOT_LOG_LEVELS) {
-      if (level == allowedRootLogLevel) {
+      if (level.equals(allowedRootLogLevel)) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Change this comparison to use the equals method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1698)